Classes/common/CDTDatastore.m

```diff
@@ -63,6 +64,7 @@ - (id)initWithDatabase:(TD_Database *)database
         NSString *name = [database name];
         _extensionsDir =
             [dir stringByAppendingPathComponent:[NSString stringWithFormat:@"%@_extensions", name]];
+        _manager = manager;
```

Comment:
There is not validation here (check the object is not nil). Does it mean that the property is optional? If that is the case, should we keep the old initialiser (`initWithDatabase:`)? Or rename the parameters this way: `- (id)initWithManager:(CDTDatastoreManager *) managerOrNil Database:(TD_Database *)database`.